### DISCUSSION TITLE
Unable To Save Files Past v1.4.3

### DISCUSSION THREAD

**Discussion by telforl** [comictagger/comictagger#378] [Q&A]
Would anybody have any idea why I cannot use any version past 1.4.3 to successfully save either .cbr or .cbz files? I'm using both the MacOS and Windows versions. I'm wondering if it's something to do with my file location, all are on a share on an Unraid server.

**Comment by lordwelch** [comment]
It would probably be related to your comics being on your share. Can you attach the log from `~/.ComicTagger/logs`? It should tell what's failing

**Comment by telforl** [comment]
[ComicTagger.log](https://github.com/comictagger/comictagger/files/9694506/ComicTagger.log)

**Reply by lordwelch** [reply]
Turns out this is the same as #376, the latest commit fixes it, I plan to put a new release out tomorrow

**Reply by telforl** [reply]
Thank you for the help and all the work you do to keep this project updated!
